instrumentation/spring-webmvc-3.1/spring-webmvc-3.1.gradle

```diff
@@ -1,31 +1,7 @@
 apply from: "$rootDir/gradle/instrumentation.gradle"
 
-muzzle {
-  pass {
-    group = 'org.springframework'
-    module = 'spring-webmvc'
-    versions = "[3.1.0.RELEASE,]"
-    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
-    skipVersions += '3.2.1.RELEASE' // missing a required class.  (bad release?)
-    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
-    assertInverse = true
-  }
-
-  // FIXME: webmvc depends on web, so we need a separate integration for spring-web specifically.
-  fail {
-    group = 'org.springframework'
-    module = 'spring-web'
-    versions = "[,]"
-    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
-    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
-  }
-}
-
 dependencies {
-  compileOnly group: 'org.springframework', name: 'spring-webmvc', version: '3.1.0.RELEASE'
-  compileOnly group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
-//  compileOnly group: 'org.springframework', name: 'spring-webmvc', version: '2.5.6'
-//  compileOnly group: 'javax.servlet', name: 'servlet-api', version: '2.4'
+  compileOnly project(':instrumentation-core:spring:spring-webmvc-3.1-core')
```

Comment:
What exactly does this auto-instrumentation modules (re)-uses from that "core" module?

Reply:
I packaged the javax.servlet-api and spring-webmvc dependencies in the core module. I thought it would be best to keep the versions in sync. I also have plans to refactor [HandlerMappingResourceNameFilter.java](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/spring-webmvc-3.1/src/main/java/io/opentelemetry/auto/instrumentation/springwebmvc/HandlerMappingResourceNameFilter.java) and [SpringWebMvcDecorator.java](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/spring-webmvc-3.1/src/main/java/io/opentelemetry/auto/instrumentation/springwebmvc/SpringWebMvcDecorator.java) and move it from the auto instrumentation module to the core module. This pull request https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/632 has these changes.

Reply:
:instrumentation-core:spring:spring-webmvc-3.1-core dependency for now